src/main/java/teammates/common/util/JsonUtils.java

```diff
@@ -78,8 +78,7 @@ public static String toJson(Object src) {
      * @see JsonParser#parse(String)
```

Comment:
Could you update the documentation as well? Thanks!

```suggestion
     * @see JsonParser#parseString(String)
```